tests/Scientist/Experiment/RunSpec.hs

```diff
@@ -62,7 +62,7 @@ spec = do
 
         let control = resultDetailsControl rd
         resultControlValue control `shouldBe` A
-        resultControlDuration control `shouldSatisfy` isDurationNear 0.100
+        resultControlDuration control `shouldSatisfy` isDurationNear 100_000_000
```

Comment:
It seems like we're saying `result.x.duration` will now be reported as whole nanoseconds instead of decimal seconds.

I'm open to that, if there's a trade-off, but in an ideal world I think this falls under compatibility with the GitHub library we're porting. To keep it consistent, this test should pass as-is with us converting the monotonic nanoseconds to decimal seconds at some point before it gets to the end-user.

That would _probably_ look like leaving the `Duration` type as-is, and converting when it's constructed at `Duration.hs#L25`. Would that undermine the goal of this PR?

Reply:
There are a few ways to approach this. Let me play around and see what works.